### DISCUSSION TITLE
What is nightly builds?

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#8302] [Q&A]
Does the nightly builds have the last update?
I mean the fixed issues and the new features till the nightly date before the last update comes?

**Comment by mergen3107** [comment]
A nightly build is usually compiled every day at a fixed time. For me it is 6 AM (US Eastern Time, convert accordingly).

You can check current commits here:
https://github.com/koreader/koreader/compare/v2021.09...master
Notice the number of commits that merged (for example, 32).

Then compare this to the actually available builds:
http://build.koreader.rocks/download/nightly/
Currently the latest is 2021-09-25, so 7 commits are still waiting for a new build.

**Reply by Frenzie** [reply]
They've been broken for the last few days because of the Let's Encrypt root certificate issue. I should have them up and running again by tomorrow.

**Reply by Frenzie** [reply]
Within some 20 minutes the new builds should be done.

**Reply by Zeyadas** [reply]
Thanks guys.
